imapserver/imapmemserver/mailbox.go

```diff
@@ -324,6 +333,10 @@ func (mbox *MailboxView) Fetch(w *imapserver.FetchWriter, numSet imap.NumSet, op
 			return
 		}
 
+		if options.ChangedSince > 0 && msg.modSeq <= options.ChangedSince {
```

Comment:
Shouldn't this be `msg.modSeq < options.ChangedSince`? In other words, we want to send the message if CHANGEDSINCE equals MODSEQ.

Reply:
The RFC specifies that the message's modSeq must be strictly greater than the CHANGEDSINCE value (we have the opposite condition)